.vuepress/config.js

```diff
@@ -105,6 +105,25 @@ module.exports = {
             '/learn/core_concepts/relevancy',
           ],
         },
+        {
+          title: '⚙️ Configuration',
+          collapsable: false,
+          path: '/learn/configuration/instance_options',
```

Comment:
@bidoubiwa: thanks to @maryamsulemani97's advice, we have realised that the errors go away if we add a `/` to the `path` values in this line and L#116. 

Is this an error on check-md/config-path-checker? `/learn/configuration/instance_options` points to a file, not a directory, so requiring a trailing slash seems like a bug. Or are we missing something?

Reply:
At the time it created issues with docs-scraper. I'm not sure it still does. Maybe we could just remove it from the CI and if we realise some pages are not indexed well we can consider putting it back. What do you think?